.gitignore

```diff
@@ -12,3 +12,6 @@ filament/docs/html/**
 gltf_baker.ini
 *tmp*.png
 civetweb.txt
+build*
```

Comment:
I'm concerned that the rules `build*` and `out*` are too general. They'll match files like `Builder.java` and `output.cpp` anywhere in the repo. Could we prepend a `/` to limit their influence to the root directory?

```
/build*
/out*
```

Reply:
Sure, thanks for the great suggestion!